java/component/keyple-calypso/src/main/java/org/eclipse/keyple/calypso/command/po/builder/ReadRecordsCmdBuild.java

```diff
@@ -14,56 +14,49 @@
 import org.eclipse.keyple.calypso.command.PoClass;
 import org.eclipse.keyple.calypso.command.po.AbstractPoCommandBuilder;
 import org.eclipse.keyple.calypso.command.po.CalypsoPoCommand;
-import org.eclipse.keyple.calypso.command.po.parser.ReadDataStructure;
 import org.eclipse.keyple.calypso.command.po.parser.ReadRecordsRespPars;
 import org.eclipse.keyple.core.seproxy.message.ApduResponse;
 
 /**
- * The Class ReadRecordsCmdBuild. This class provides the dedicated constructor to build the Read
+ * The {@link ReadRecordsCmdBuild} class provides the dedicated constructor to build the Read
  * Records APDU command.
  */
 public final class ReadRecordsCmdBuild extends AbstractPoCommandBuilder<ReadRecordsRespPars> {
 
-    /** The command. */
     private static final CalypsoPoCommand command = CalypsoPoCommand.READ_RECORDS;
 
-    /* Construction arguments */
+    // Construction arguments used for parsing
     private final int sfi;
-    private final byte firstRecordNumber;
-    private final ReadDataStructure readDataStructure;
+    private final int firstRecordNumber;
+    private final boolean readJustOneRecord;
```

Comment:
We could define an inner Enum ReadMod ONE_RECORD MULTIPLE_RECORD

Reply:
planned for the next PR